server/ladder_service.py

```diff
@@ -77,19 +77,6 @@ async def initialize(self) -> None:
 
     async def update_data(self) -> None:
         async with self._db.acquire() as conn:
-            # Legacy ladder1v1 map pool
-            # TODO: Remove this https://github.com/FAForever/server/issues/581
-            result = await conn.execute(
-                "SELECT ladder_map.idmap, "
-                "table_map.name, "
-                "table_map.filename "
-                "FROM ladder_map "
-                "INNER JOIN table_map ON table_map.id = ladder_map.idmap"
-            )
-            maps = [Map(*row.as_tuple()) async for row in result]
-
-            self.ladder_1v1_map_pool.set_maps(maps)
```

Comment:
We can remove `ladder_1v1_map_pool` completely from `LadderService`. Which means we can also remove the hardcoded `ladder1v1` queue and the code that initializes it (line 73).

Reply:
Ohh yes I see, removed that as well.